maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svntest/src/main/java/org/apache/maven/scm/provider/svn/command/blame/SvnBlameCommandTckTest.java

```diff
@@ -27,6 +27,8 @@
 import java.io.File;
 import java.util.List;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
I don't understand why this import is added; it is the only change in this file?

Reply:
In junit3 you inherited this method (w/ same signature from TestCase), but in Junit4 you need to use hamcrest, or Assert or alike, hence static import but no other change (as method sig is same).